specification/schema/content.schema.json

```
@@ -18,6 +18,18 @@
             "type": "string",
             "description": "A uri that points to tile content. When the uri is relative, it is relative to the referring tileset JSON file."
         },
+        "metadata": {
+            "allOf": [
+                {
+                    "$ref": "metadataEntity.schema.json"
+                }
+            ],
+            "description": "Content metadata."
+        },
+        "group": {
+            "type": "string",
+            "description": "The group this content belongs to. The value must correspond to a group ID declared in the `groups` dictionary."
```

The `draft-1.1-schema` branch was out of date here. It should now be:

```suggestion
            "description": "The group this content belongs to. The value is an index into the array of `groups` that is defined for the containing tileset."
```